core/sql/executor/HiveClient_JNI.cpp

```diff
@@ -68,6 +68,27 @@ char* HiveClient_JNI::getErrorText(HVC_RetCode errEnum)
     return (char*)hvcErrorEnumStr[errEnum-HVC_FIRST-1];
 }
 
+//////////////////////////////////////////////////////////////////////////////
+// 
+//////////////////////////////////////////////////////////////////////////////
+HiveClient_JNI* HiveClient_JNI::newInstance(NAHeap *heap, HVC_RetCode &retCode)
+{
+   QRLogger::log(CAT_SQL_HDFS, LL_DEBUG, "HiveClient_JNI::newInstance() called.");
```

Comment:
Just curious: Why do we use CAT_SQL_HDFS in this QRLogger::log call, but elsewhere in this module we use CAT_SQL_HBASE? (Perhaps this use is correct and the others are all wrong?)

Reply:
Let me try to get rid of logging unless it makes sense